scripts/es_snapshot.py

```diff
@@ -249,8 +252,8 @@ def run(self):
         for version in live_snapshots:
             print(f'    {version}')
 
-        confirm = input(
-            f"Are you sure you want to delete all files that don't belong to a live ES snapshot? [y/n]")
+        confirm = eval(input(
```

Comment:
why would we want to `eval` this? That seems very dangerous

Reply:
@AmitPhulera I think (confusingly) this file is already a py3-only file that is meant to be copied directly to a server any run on that server with its own `python3` executable, something that we get a hint at from the first line in the file
```
#!/usr/bin/env python3
```

So I agree with Simon. I think all of the changes in this file should be reverted so that the PR doesn't touch this file at all. That should also be true about any file whose first line matches `'^\#\!.*python3'`, which you can quickly find with `grep -R '^\#\!.*python3' .`

Reply:
There were only two such files
- `es_snapshot.py`
- `pg_repack.py`

Reverted both of them back.